demo/lightning/image-segmentation/train.py

```diff
@@ -48,15 +60,16 @@ def init_processes():
     if not flags.local:
         init_processes()
     profiler = None
+    callbacks = []
     if flags.benchmark:
         profiler = "simple"
+        callbacks = [EpochTimer()]
```

Comment:
```suggestion
        callbacks.append(EpochTimer())
```
nitpick: It would be better to append to `callbacks` instead of replacing it in case we have future enhancements that add additional callbacks.